app/backend/approaches/retrievethenread.py

```diff
@@ -118,7 +118,7 @@ async def run(
                 # Azure Open AI takes the deployment name as the model name
                 model=self.chatgpt_deployment if self.chatgpt_deployment else self.chatgpt_model,
                 messages=message_builder.messages,
```

Comment:
I sometimes see issues related to "output isn't consistent". In a follow up PR would love to see us using the seed parameter and let customers put it in developer settings

https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/reproducible-output?tabs=pyton